qemu/tests/qemu_io_blkdebug.py

```diff
@@ -92,8 +92,8 @@ def run(test, params, env):
         else:
             session.close()
             try:
-                image_io.snapshot_del(blkdebug_cfg=blkdebug_cfg)
-                output = ""
+                cmd_result = image_io.snapshot_del(blkdebug_cfg=blkdebug_cfg)
+                output = cmd_result.stdout + cmd_result.stderr
```

Comment:
@PandaWei yes. but that patch is always for disable the exception triggered by process.run(). I just update it to reduce impact for positive test.